imap/command.c

```diff
@@ -440,6 +440,7 @@ static int cmd_status (const char *s)
 /* cmd_handle_fatal: when IMAP_DATA is in fatal state, do what we can */
 static void cmd_handle_fatal (IMAP_DATA* idata)
 {
+  static int try_reconnect = 1;
```

Comment:
The function *is* reentrant, just it doesn't behave the same once reentred ;) I don't see a benefit to moving this local piece of data to IMAP_DATA. What am I missing?